scripts/package/package-bcd.js

```diff
@@ -0,0 +1,12 @@
+const bcd = require('mdn-browser-compat-data');
+
+function package(query) {
```

Comment:
https://github.com/mdn/browser-compat-data/issues/4106

Reply:
> how many times I've seen this  <strike>implemented</strike> copy/pasted

Reply:
One first impression: it is long past time for BCD to provide this API. It's wild how many times I've seen this implemented 😆